include/alpaka/test/event/EventHostManualTrigger.hpp

```diff
@@ -1,4 +1,5 @@
-/* Copyright 2023 Benjamin Worpitz, Matthias Werner, Jan Stephan, Andrea Bocci, Bernhard Manfred Gruber
+/* Copyright 2023 Benjamin Worpitz, Matthias Werner, Jan Stephan, Jeffrey Kelling, Andrea Bocci,
+ *                Bernhard Manfred Gruber
```

Comment:
No other changes to the file.

Reply:
The only change (apart from clang format I suppose) is Jeffrey Kelling, who made [changes to the file](https://github.com/Parsifal-2045/alpaka/commit/e4cddb74b47869dc634a369cc99fbd5f84b99ed5).

Reply:
The copyright headers are on me. I added file authors from the git history where they were missing.

Reply:
Alright then. So feel free to ignore any comments in this regard then.